lua/resession/init.lua

```diff
@@ -265,7 +265,7 @@ local function save(name, opts, target_tabpage)
   for ext_name, ext_config in pairs(config.extensions) do
     local ext = util.get_extension(ext_name)
     if ext and ext.on_save and (ext_config.enable_in_tab or not target_tabpage) then
-      local ok, ext_data = pcall(ext.on_save)
+      local ok, ext_data = pcall(ext.on_save, target_tabpage)
```

Comment:
Would prefer if `target_tabpage` was part of an `opts` object. Makes it easier to extend later if we want to pass in more data.

Reply:
That makes sense, I'll update this later today. I'll update docs & type annotions then as well, realized I forgot to do that.